gigaword-indexer/src/main/java/edu/isi/vista/gigawordIndexer/ConcatenatedGigawordDocuments.java

```diff
@@ -36,7 +36,7 @@ public static ConcatenatedGigawordDocuments fromGigwordGZippedFile(Path p) throw
     private String id;
     private String text;
 
-    private Article(String id, String text) {
```

Comment:
Now that it is being used in two places (and soon we will be adding a third), can you promote this to a top-level class?

Reply:
Fixed in newest commit.